packages/compartment-mapper/src/import-hook.js

```diff
@@ -45,13 +45,27 @@ function getImportsFromRecord(record) {
   return (has(record, 'record') ? record.record.imports : record.imports) || [];
 }
 
+// Node.js resolution allows for an incomplement specifier.
```

Comment:
“incomplete”? Can you add an example of one?

Reply:
as in "lacking a suffix", like `./util`. maybe ill just write that